source_gen/test/builder_test.dart

```diff
@@ -344,6 +344,31 @@ part "a.foo.dart";'''
     });
   });
 
+  group('LibraryBuilder', () {
+    group('with build_extensions', () {
+      test('generates relative `path of` for output in different directory',
```

Comment:
This name doesn't seem quite right, maybe:
```suggestion
      test('outputs to the correct location',
```

Reply:
whoops